controllers/provisioners/eks/eks.go

```diff
@@ -34,13 +35,17 @@ const (
 	OsFamilyAnnotation                  = "instancemgr.keikoproj.io/os-family"
 	ClusterAutoscalerEnabledAnnotation  = "instancemgr.keikoproj.io/cluster-autoscaler-enabled"
 
-	OsFamilyWindows                     = "windows"
+	OsFamilyWindows      = "windows"
+	OsFamilyBottleRocket = "bottlerocket"
```

Comment:
Should we add amazonlinux2 annotation and make it the default value?
e.g. in validation of the API (instancegroups_types.go) can add this annotation if it's missing? or should we just assume that if this annotation is missing, it's linux - WDYT?

Reply:
Yeah, that probably makes sense.

Reply:
Added this.